core/src/mindustry/world/blocks/Autotiler.java

```diff
@@ -14,27 +14,53 @@
 //TODO documentation
 public interface Autotiler{
 
-    //holds some static temporary variables, required due to some RoboVM bugs
+    /**
+     * Holds some static temporary variables, required due to some RoboVM bugs
+     */
     class AutotilerHolder{
         static final int[] blendresult = new int[5];
         static final BuildPlan[] directionals = new BuildPlan[4];
     }
 
-    /** slices a texture region:
-     * mode == 0 -> no slice
-     * mode == 1 -> bottom
-     * mode == 2 -> top */
-    default TextureRegion sliced(TextureRegion input, int mode){
-        return mode == 0 ? input : mode == 1 ? botHalf(input) : topHalf(input);
+    /**
+     * The mode to slice a texture at.
+     */
+    enum SliceMode{
+        NO_SLICE,
```

Comment:
Never use `SCREAMING_SNAKE_CASE`, see [the guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md#follow-the-formatting-guidelines) for why.

Reply:
Ok, ill fix that now